helm/kvm-operator/templates/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       annotations:
         releaseRevision: {{ .Release.Revision | quote }}
       labels:
-        {{- include "labels.selector" . | nindent 8 }}
+        {{- include "labels.common" . | nindent 8 }}
```

Comment:
This comes from https://github.com/giantswarm/aws-operator/pull/2451 This allows us to match the pods in okteto based on branch label and also brings kvm-operator in sync with aws-operator's SOTA. It only adds new labels without changing existing ones so I don't expect this to affect anything.